### DISCUSSION TITLE
ogb __init__.py file issues when running from terminal

### DISCUSSION THREAD

**Discussion by DenisTurcu** [snap-stanford/ogb#339] [General]
Hello,

I encountered a weird issue when running most of the provided examples from the terminal. The issue does not appear when copying and accordingly adjusting the respective main*.py files code into Jupyter notebooks and running from there, or when running in debug mode.

The issue is that, when running from the terminal as instructed (e.g. "python main_pyg.py" for graph prediction on ogbg-molhiv), the program stalls and does nothing. After many hours of tracing the problem, I have found that commenting or emptying the ogb/__init__.py file solves this issue, but then I do not have access to ogb.__version__ (not a big problem, though). The new ogb/__init__.py file needs to be **empty**, meaning I had to delete the following line:

`from .version import __version__`

Anyone else having this weird issue? I installed from scratch on both Ubuntu and Mac on different computers and got the same problem. Any suggestions how to fix it properly?

**Comment by rusty1s** [comment]
Might be related to https://github.com/snap-stanford/ogb/issues/322.
